test/export/StructureDefinitionExporter.test.ts

```diff
@@ -343,6 +343,32 @@ describe('StructureDefinitionExporter', () => {
     expect(exported.derivation).toBe('constraint');
   });
 
+  it('should export sub-extensions, with similar starting names and different types', () => {
```

Comment:
Once this test is correctly failing, can you skip it using `it.skip`? That way we can merge in the failing test to master before pushing up the fix. Then, when we have a bug fix ready, we can un-skip the test.